pkg/dock/dock.go

```diff
@@ -419,16 +464,21 @@ func (ds *dockServer) UpdateVolumeGroup(ctx context.Context, opt *pb.UpdateVolum
 
 func (ds *dockServer) DeleteVolumeGroup(ctx context.Context, opt *pb.DeleteVolumeGroupOpts) (*pb.GenericResponse, error) {
 	// Get the storage drivers and do some initializations.
-	ds.Driver = drivers.Init(opt.GetDriverName())
-	defer drivers.Clean(ds.Driver)
+	driver, err := drivers.Init(opt.GetDriverName())
```

Comment:
Why use local variable `driver` instead of `ds.Driver`

Reply:
cause ds.Driver is a global variable, each time drivers.Init will generate a new driver object and assign to it.
this has problem for concurrency scenario, the driver generated by one thread may be covered by another one.